opensrp-reveal/src/test/java/org/smartregister/reveal/view/ListTasksActivityTest.java

```diff
@@ -0,0 +1,716 @@
+        Whitebox.setInternalState(listTasksActivity, "listTaskPresenter", listTaskPresenter);
+        listTasksActivity.findViewById(R.id.btn_edit_paot_details).performClick();
+        verify(listTaskPresenter).onChangeInterventionStatus(Intervention.PAOT);
+    }
+
+    @Test
+    public void testCloseCardView() {
+
+        listTasksActivity.findViewById(R.id.btn_collapse_spray_card_view).performClick();
+        assertEquals(GONE, listTasksActivity.findViewById(R.id.spray_card_view).getVisibility());
+    }
+
+    @Test
+    public void testOtherCardView() {
+        listTasksActivity.findViewById(R.id.btn_collapse_mosquito_collection_card_view).performClick();
+        assertEquals(GONE, listTasksActivity.findViewById(R.id.mosquito_collection_card_view).getVisibility());
+    }
+
+    @Test
+    public void testOpenDrawerMenu() {
+        Whitebox.setInternalState(listTasksActivity, "drawerView", drawerView);
+        listTasksActivity.findViewById(R.id.drawerMenu).performClick();
+        verify(drawerView).openDrawerLayout();
+    }
+
+    @Test
+    public void testOpenIndicators() {
+        assertEquals(GONE, listTasksActivity.findViewById(R.id.indicators_card_view).getVisibility());
+        listTasksActivity.findViewById(R.id.progressIndicatorsGroupView).performClick();
+        assertEquals(VISIBLE, listTasksActivity.findViewById(R.id.indicators_card_view).getVisibility());
+    }
+
+    @Test
+    public void testRegisterFamily() {
+        Whitebox.setInternalState(listTasksActivity, "listTaskPresenter", listTaskPresenter);
+        when(listTaskPresenter.getSelectedFeature()).thenReturn(feature);
+        listTasksActivity.findViewById(R.id.register_family).performClick();
+        Intent startedIntent = shadowOf(listTasksActivity).getNextStartedActivity();
+        assertEquals(FamilyRegisterActivity.class, shadowOf(startedIntent).getIntentClass());
+        assertEquals(GONE, listTasksActivity.findViewById(R.id.spray_card_view).getVisibility());
+        assertTrue(startedIntent.hasExtra(TASK_IDENTIFIER));
+    }
+
+
+    @Test
+    public void testOpenStructureProfile() {
+        Whitebox.setInternalState(listTasksActivity, "listTaskPresenter", listTaskPresenter);
+        when(listTaskPresenter.getSelectedFeature()).thenReturn(feature);
+        CommonPersonObjectClient client = TestingUtils.getCommonPersonObjectClient();
+        listTasksActivity.openStructureProfile(client);
+        Intent startedIntent = shadowOf(listTasksActivity).getNextStartedActivity();
+        assertEquals(FamilyProfileActivity.class, shadowOf(startedIntent).getIntentClass());
+        assertEquals(client.getColumnmaps().get(FIRST_NAME), startedIntent.getStringExtra(INTENT_KEY.FAMILY_NAME));
+        assertEquals(client.getCaseId(), startedIntent.getStringExtra(INTENT_KEY.FAMILY_BASE_ENTITY_ID));
+        assertTrue(startedIntent.hasExtra(TASK_IDENTIFIER));
+
+    }
+
+
+    @Test
+    public void testSetGeoJsonSource() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4684568)